db/models/review.js

```diff
@@ -0,0 +1,38 @@
+'use strict'
+
+const {STRING, TEXT, INTEGER} = require('sequelize')
+
+module.exports = db => db.define('reviews', {
+	content: {
+		type: TEXT,
+		defaultValue: '',
```

Comment:
Similar thing as before, I think the empty string default value is not great. Why not have `allowNull: true`  ?
  We would not be able to get all reviews that have text?   I am assuming you want the user to just be able to give review without text.